src/MongoDB/Session.c

```diff
@@ -98,6 +98,63 @@ static const char* php_phongo_get_transaction_state_string(mongoc_transaction_st
 	}
 }
 
+static void php_phongo_transaction_options_to_zval(mongoc_client_session_t* cs, zval* retval)
+{
+	mongoc_transaction_opt_t*     opts;
+	int64_t                       max_commit_time_ms;
+	const mongoc_read_concern_t*  read_concern;
+	const mongoc_read_prefs_t*    read_preference;
+	const mongoc_write_concern_t* write_concern;
+
+	if (!cs) {
+		ZVAL_NULL(retval);
+		return;
+	}
+
+	opts = mongoc_session_opts_get_transaction_opts(cs);
+
+	if (!opts) {
+		ZVAL_NULL(retval);
+		return;
+	}
+
+	max_commit_time_ms = mongoc_transaction_opts_get_max_commit_time_ms(opts);
+	read_concern       = mongoc_transaction_opts_get_read_concern(opts);
+	read_preference    = mongoc_transaction_opts_get_read_prefs(opts);
+	write_concern      = mongoc_transaction_opts_get_write_concern(opts);
+
+	array_init_size(retval, 4);
+
+	if (max_commit_time_ms) {
+		ADD_ASSOC_LONG_EX(retval, "maxCommitTimeMS", max_commit_time_ms);
+	}
+
+	if (!mongoc_read_concern_is_default(read_concern)) {
+		zval zread_concern;
+
+		phongo_readconcern_init(&zread_concern, read_concern);
+		ADD_ASSOC_ZVAL_EX(retval, "readConcern", &zread_concern);
+	}
+
+	if (read_preference) {
+		zval zread_preference;
+
+		phongo_readpreference_init(&zread_preference, read_preference);
+		ADD_ASSOC_ZVAL_EX(retval, "readPreference", &zread_preference);
+	}
+
+	if (!mongoc_write_concern_is_default(write_concern)) {
+		zval zwrite_concern;
+
+		phongo_writeconcern_init(&zwrite_concern, write_concern);
+		ADD_ASSOC_ZVAL_EX(retval, "writeConcern", &zwrite_concern);
+	}
+
+	if (opts) {
+		mongoc_transaction_opts_destroy(opts);
```

Comment:
I believe this change was made through the "Refactor php_phongo_transaction_options_to_zval, free the mongoc_transaction_opt_t" commit. Please amend that commit to reference PHPC-1879 so we can ensure it gets linked up in JIRA when this is ultimately merged. You'll also want to ensure that we keep that line in the final message when squashing/merging the PR.

I suppose we could just wait until merge time to add a reference to PHPC-1879, but this would give you a good reason to learn how to use `git rebase` to change messages of existing commits (among other things). See https://docs.github.com/en/github/committing-changes-to-your-project/creating-and-editing-commits/changing-a-commit-message#amending-older-or-multiple-commit-messages for docs on that.